src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-post.ts

```diff
@@ -33,13 +33,11 @@ export const JiraConnectEnterpriseAppPost = async (
 			installationId: installation.id
 		});
 
-		res.status(201).json({
-			message: `Successfully added GitHub Server App for ${installation.id}`,
-			data: githubServerApp
-		});
+		res.status(200).send({ success: true, data: githubServerApp });
```

Comment:
Updated this to align with how we are sending other responses in the server flow.